pynsee/localdata/get_geo_list.py

```diff
@@ -94,7 +93,7 @@ def get_geo_list(geo=None, date=None, update=False, silent=False):
                     date=date,
                     type=type_geo,
                 )
-            except RequestException:
+            except ValueError:

```

Comment:
Could you specify why/where the value error comes from in the ``_get_geo_relation``? (maybe adding a comment there?)
I suspect you ran into the error and tracked it down but it's really not obvious from reading the function, so having it explicitely written in the docstring would be nice.

Reply:
After some thought, I'm not sure that I understand that comment 🙁 .

The primary object of this PR was to ensure unity of exceptions' types generated by `_request_insee` (whether we're using a token or not, whether we use api_url or sdmx_url). As `get_geo_list` runs `_get_geo_relation` which runs `_request_insee`, the exception's type should be the same.

I've just added a simple comment and a docstring to _get_geo_relation (which was missing indeed) , was this what you had in mind?

Reply:
that's fine, this indeed answers my question